lib/administrate/search.rb

```diff
@@ -88,9 +88,10 @@ def query_template
           attribute_type = attribute_types[attr]
 
           search_target = "#{table_name}.#{column_name}"
-          search_target = "LOWER(CAST(#{search_target} AS CHAR(256)))" if attribute_type.search_lower?
+          search_target = "CAST(#{search_target} AS CHAR(256))" unless attribute_type.search_exact?
```

Comment:
`search_exact` is a new option that's only used by the text message and email dashboards (not visible to non super-admins)